### PR TITLE
Add optimized code for speeding up get recent videos endpoint

### PR DESCRIPTION
**Pull request by prakharrathi25**
This PR alters the `get_recent` endpoint in the `video` app to optimize the speed of fetching the videos. 

It also adds more choices in the `LANGUAGE_CHOICE` constant in `translation\metadata`. 

The speedup result has been mentioned in the [JIRA ticket](https://project-sunbird.atlassian.net/browse/SN-518?atlOrigin=eyJpIjoiMjkzMDI0NTI4NjQ0NDZlMmJhMjJjMDY5Nzc3YWZlNjYiLCJwIjoiaiJ9).

### CODE REVIEW

File: backend/video/views.py [hunk 1]
```diff
@@ -182,16 +182,29 @@ def list_recent(request):
...
         (transcript.video, transcript.updated_at)
-        for transcript in Transcript.objects.filter(user=request.user.id)
+        for transcript in Transcript.objects.filter(user=request.user.id).order_by(
+            "-updated_at"
+        )[:count]
     ]
 
-    # Get a list of videos from recently translated videos
+    # Get the date of the first trancript from the above list
+    first_transcript_date = recent_transcripts[-1][1]
```
Comment (GokulNC) [inline backend/video/views.py:199]: "last" transcript? Because code says `[-1]`

File: backend/video/views.py [hunk 2]
```diff
@@ -182,16 +182,29 @@ def list_recent(request):
     # Get the query param from the request, default count is 10
     count = int(request.query_params.get("count", 10))
 
-    # Get a list of videos from recently transcribed videos
+    # Note: Currently, we have implemented this get recent method based on the logic that
+    # one Transcript of either type ORIGINAL_SOURCE or type MACHINE_GENERATED
+    # will always have one video associated with it.
+    # In the future, if that constraint is removed then we might need to alter the logic.
+
+    # Get the 2n latest transcripts from the DB for the user
```
Comment (GokulNC) [inline backend/video/views.py:190]: It's `1n` for now. In the future, if the above constraint is removed, we may have to make it `2n`

File: backend/video/views.py [hunk 3]
```diff
@@ -182,16 +182,29 @@ def list_recent(request):
...
+        for transcript in Transcript.objects.filter(user=request.user.id).order_by(
+            "-updated_at"
+        )[:count]
     ]
 
-    # Get a list of videos from recently translated videos
+    # Get the date of the first trancript from the above list
+    first_transcript_date = recent_transcripts[-1][1]
+
+    # Get the 2n latest translations from the DB for the user
```
Comment (GokulNC) [inline backend/video/views.py:201]: We are not doing this top-`2n` thing for translations now, right?

**Review by GokulNC — APPROVED** [review]
Looks good to me, thanks!

Please just make these minor changes to the comments so that it's clear.

File: backend/video/views.py (hunk 1 above)
Comment (prakharrathi25) [reply backend/video/views.py:199]: So, it's sorted in descending order. Basically, we want the least recently updated transcript in the top 10 recently updated transcript list.

File: backend/video/views.py (hunk 2 above)
Comment (prakharrathi25) [reply backend/video/views.py:190]: Yes, I will update that.

File: backend/video/views.py (hunk 3 above)
Comment (prakharrathi25) [reply backend/video/views.py:201]: Will remove.

File: backend/video/views.py (hunk 1 above)
Comment (prakharrathi25) [reply backend/video/views.py:199]: I have updated the variable names to make them a bit more clear.

**Comment by prakharrathi25** [comment]
I am merging after resolving all comments.

**Pull request merged**